md_main.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import logging
+import pandas as pd
+import pathlib
+import shutil
+import sqlite3
+from os import path
+
+from scripts.database_operations import *
```

Comment:
It is better to import specific entities, it avoids some potentially confusing bugs that can arise